src/node/util.ts

```diff
@@ -14,15 +14,16 @@ export interface FrontmatterTempMap {
 export function curryFrontmatterHandler(
   scope: string,
   map: FrontmatterTempMap,
+  path: string,
 ): FrontmatterHandler
-export function curryFrontmatterHandler(scope, map) {
+export function curryFrontmatterHandler(scope, map, path) {
   return (key, pageKey) => {
     if (key) {
       if (!map[key]) {
         map[key] = {}
         map[key].key = key
         map[key].scope = scope
-        map[key].path = `/${scope}/${key}/`
+        map[key].path = path ? `${path}${key}` : `/${scope}/${key}/`
```

Comment:
1. Since `path` is required, the condition will never be false.
2. The slash in the end is missing.
```suggestion
        map[key].path = `${path}${key}/`
```